src/views/queriesList.tsx

```diff
@@ -0,0 +1,124 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { ButtonIcon, Page } from './common.js'
+
+type QueryStatus = 'resolved' | 'pending_your_input' | 'pending_their_input'
+
+interface Query {
+  company_name: string
+  query_type: string
+  direction: 'sent' | 'received'
+  updated_at: Date
+  status: QueryStatus
+}
+@singleton()
+export default class QueryListTemplates {
+  constructor() {}
+
+  private statusToClass = (status: string | QueryStatus): JSX.Element => {
+    switch (status) {
+      case 'pending_your_input':
+        return <div class="warning">Pending Your Input</div>
+      case 'pending_their_input':
+        return <div class="disconnected">Pending Their Input</div>
+      case 'resolved':
+        return <div class="success">Resolved</div>
+      default:
+        return <div class="error">unknown</div>
+    }
+  }
+
+  public listPage = (queries: Query[], search: string = '') => {
+    return (
+      <Page
+        title="Veritable - Queries"
+        heading="Query Management"
+        headerLinks={[{ name: 'Query Management', url: '/query-management' }]}
+      >
+        <div
+          class="main connections"
```

Comment:
There's a bunch of `connection` related stuff in this file that I assume are hangovers from the original table. You have two choices:

1. generify the existing classes/ids so they are not connection specific. This may involve moving things to shared stylesheets. It may also involve extracting common utility classes
2. rename elemeents and classes in this file to be `query` rather than `connection` related. This will then involve creating new CSS which may be a lot of copying but allows us to maintain them separately

The ideal solution is likely a mixture of the two approaches

Reply:
yh I'll have a look .. I thought this could be done in the `fixing-css` ticket, but I'll do it here